docs/exercises/5-multi_region.rst

```diff
@@ -204,13 +207,13 @@ we won't need to keep around the key ID, so we can discard that value.
 
     .. group-tab:: Python
 
-        We won't need to keep the key ID around, so replace that in ``__init__`` with a call to the new
-        multiple master key provider constructor.
+        We will be constructing a new multi region KMS Master Key Provider, so replace the call to the
```

Comment:
Minor: multi-CMK rather than multi-region? The CMKs are incidentally in a different region, but it's not a required property

Reply:
Will be pushing a commit that fixes this as well